src/global_vars.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+#include "i2c.h"
+
+#define delay(ms) do { \
+    for (int i = 0; i < ms * 2000; i++) \
+        __asm__( \
```

Comment:
```suggestion
        __asm__( \
        // 30 x nop
```

Reply:
I've added a comment explaining this above the define.